NewHorizons/Components/StarLightController.cs

```diff
@@ -96,6 +112,19 @@ public void Update()
                     break;
                 }
             }
+
+            if (PlayerState.InMapView())
+            {
+                foreach (var light in _lights) light.enabled = true;
+                return;
```

Comment:
Instead of returning here just just put what comes next inside an `else`, makes it clearer